stories/topics/ref-saas-planning-configs.adoc

```diff
@@ -0,0 +1,5 @@
+[id="ref-saas-planning-configs"]
+= Planning the required configurations for {SaaSonAWSShort}
+After accessing your {PlatformNameShort} instance, you can confirm that there is only a control node, and mesh ingress hop nodes configured on the *Instances* page or *Topology* page. 
```

Comment:
You configure the nodes on the Create Instances page and view the results using the Topology viewer.

Reply:
Hey Ian, appreciate  the callout here, but just confirming via SaaS the nodes are configured automatically (after the user submits the registration form on AWS), this step is just telling the user where to look to confirm for themselves, they won't actually need to configure these. 

Maybe if this sentence is confusing I could say something like:
"you can confirm that there is only a control node, and mesh ingress hop nodes on the *Instances* page or *Topology* page. "

Reply:
That would work.